BUILD.md

```diff
@@ -248,10 +238,10 @@ can run the following commands in the `dz` app qube:
 poetry shell
 
 # run the CLI
-./dev_scripts/dangerzone-cli --help
+VM_CONVERSION=1 ./dev_scripts/dangerzone-cli --help
```

Comment:
Given that we follow Qubes terminology (where VM -> Qube), shouldn't we do something similar here? I.e., `QUBES_CONVERSION` instead of `VM_CONVERSION`?

Reply:
Good point. Addressed in [dd96de3](https://github.com/freedomofpress/dangerzone/pull/464/commits/dd96de32eb181b6a954c35f08fa2b31335be2008)